Sanya_form.HTML

```diff
@@ -34,21 +34,21 @@
       <input type="email" placeholder="email"/>
         
         <br/>
-        
-      <b><label>Gender:</label></b>
-
-        <br/>
 
       <b><label>Mobile:</label></b>
       <input type="mobile" />
```

Comment:
"mobile" is not a type.  There is a type that you can use for telephone numbers though